luaui/Widgets/gui_gridmenu.lua

```diff
@@ -1992,17 +2173,15 @@ local function handleButtonHover()
 	if not WG["topbar"] or not WG["topbar"].showingQuit() then
 		if hovering then
 			for cellRectID, cellRect in pairs(cellRects) do
-				if cellRect:contains(x, y) then
+				if cellRect.opts.uDefID and cellRect:contains(x, y) then
 					hoveredCellID = cellRectID
-					local cmd = cellCmds[cellRectID]
-					local uDefID = -cmd.id
+					local uDefID = cellRect.opts.uDefID
```

Comment:
Does "hoveredButtonNotFound" need to be set to false somewhere in the cell rect checks?

Reply:
Not sure what that means... The variable is used as a temporary check that any button has been hovered and used later. In any manner I am not touching handle button hover for now, the whole thing needs to be rewritten and extracted so state gets set somewhere else.

Reply:
The concern is relevant, but I'll tackle it either on a next PR or next iteration

Reply:
All this garbage has been removed, the check is somewhat different now.